File: TvShowServer/Model/show.py
class Show(object):
    def __init__(self,id,name,poster,overview,type,language,seasons,episodes,runtime,status,airDate,popularity,votes,backdrop):
        self.id = id
        self.name = name
        self.poster = poster
        self.overview = overview
        self.type = type
        self.language = language
        self.seasons = seasons
        self.episodes = episodes
        self.runtime = runtime
        self.status = status
        self.airDate = airDate
        self.popularity = popularity
        self.votes = votes
        self.backdrop = backdrop
        self.networks = ''
        self.creators = ''
        self.genres = ''
        self.actors = ''

    def getId(self):
        return self.id

    def getName(self):
        return self.name

    def getPoster(self):
        return self.poster

    def getOverview(self):
        return self.overview

    def getType(self):
        return self.type

    def getLanguage(self):
        return self.language

    def getSeasons(self):
        return self.seasons

    def getEpisodes(self):
        return self.episodes

    def getRuntime(self):
        return self.runtime

    def getStatus(self):
        return self.status

    def getAirDate(self):
        return self.airDate

    def getPopularity(self):
        return self.popularity

    def getVotes(self):
        return self.votes

    def getBackdrop(self):
        return self.backdrop

    def getActors(self):
        return self.actors

    def getCreators(self):
        return self.creators

    def getGenres(self):
        return self.genres

    def getNetworks(self):
        return self.networks

    def setActors(self,actors):
        self.actors = actors

    def setCreators(self, creators):
        self.creators = creators

    def setGenres(self, genres):
        self.genres = genres

    def setNetworks(self, networks):
        self.networks = networks

    def serialize(self):
        return {'id': self.id,
                'name': self.name,
                'poster': self.poster,
                'overview': self.overview,
                'type': self.type,
                'language': self.language,
                'seasons': self.seasons,
                'episodes': self.episodes,
                'runtime': self.runtime,
                'status': self.status,
                'airDate': self.airDate,
                'popularity': self.popularity,
                'votes': self.votes,
                'backdrop': self.backdrop,
                'networks': self.networks,
                'creators': self.creators,
                'genres': self.genres,
                'actors': self.actors
                }

File: TvShowApp/app/src/main/java/com/gm/gislain/tvshowapp/Model/Show.java
package com.gm.gislain.tvshowapp.Model;

import java.util.ArrayList;

/**
 * Created by Gislain on 25/11/2016.
 */

public class Show {

    private int id;
    private String name;
    private String poster;
    private String overview;
    private String type;
    private String language;
    private Integer seasons;
    private Integer episodes;
    private Integer runtime;
    private String status;
    private String date;
    private double popularity;
    private double votes;
    private String backdrop;
    private ArrayList<Creator> creators;
    private ArrayList<Actor> actors;
    private ArrayList<Network> networks;
    private ArrayList<Genre> genres;

    public static String KEY_ID = "id";
    public static String KEY_NAME = "name";
    public static String KEY_POSTER = "poster";
    public static String KEY_OVERVIEW = "overview";
    public static String KEY_TYPE = "type";
    public static String KEY_LANGUAGE = "language";
    public static String KEY_SEASONS = "seasons";
    public static String KEY_EPISODES = "episodes";
    public static String KEY_RUNTIME = "runtime";
    public static String KEY_STATUS = "status";
    public static String KEY_DATE = "airDate";
    public static String KEY_POPULARITY = "popularity";
    public static String KEY_VOTES = "votes";
    public static String KEY_ACTORS = "actors";
    public static String KEY_CREATORS = "creators";
    public static String KEY_NETWORKS = "networks";
    public static String KEY_GENRES = "genres";
    public static String KEY_BACKDROP = "backdrop";

    public Show() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPoster() {
        return poster;
    }

    public void setPoster(String poster) {
        this.poster = poster;
    }

    public String getOverview() {
        return overview;
    }

    public void setOverview(String overview) {
        this.overview = overview;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getLanguage() {
        return language;
    }

    public void setLanguage(String language) {
        this.language = language;
    }

    public Integer getSeasons() {
        return seasons;
    }

    public void setSeasons(Integer seasons) {
        this.seasons = seasons;
    }

    public Integer getEpisodes() {
        return episodes;
    }

    public void setEpisodes(Integer episodes) {
        this.episodes = episodes;
    }

    public Integer getRuntime() {
        return runtime;
    }

    public void setRuntime(Integer runtime) {
        this.runtime = runtime;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public double getPopularity() {
        return popularity;
    }

    public void setPopularity(double popularity) {
        this.popularity = popularity;
    }

    public double getVotes() {
        return votes;
    }

    public void setVotes(double votes) {
        this.votes = votes;
    }

    public String getBackdrop() {
        return backdrop;
    }

    public void setBackdrop(String backdrop) {
        this.backdrop = backdrop;
    }

    public ArrayList<Creator> getCreators() {
        return creators;
    }

    public void setCreators(ArrayList<Creator> creators) {
        this.creators = creators;
    }

    public ArrayList<Actor> getActors() {
        return actors;
    }

    public void setActors(ArrayList<Actor> actors) {
        this.actors = actors;
    }

    public ArrayList<Network> getNetworks() {
        return networks;
    }

    public void setNetworks(ArrayList<Network> networks) {
        this.networks = networks;
    }

    public ArrayList<Genre> getGenres() {
        return genres;
    }

    public void setGenres(ArrayList<Genre> genres) {
        this.genres = genres;
    }
}
